src/main/java/frc/team3388/robot/SystemFactory.java

```diff
@@ -31,6 +33,15 @@ public ExampleSystem createExampleSystem() {
         return new ExampleSystem(motor);
     }
 
+
+    public ShooterSystem createShooterSystem() {
```

Comment:
You kind of missed  the comment earlier, so I'll do it here.

You need to create an instance of your system as well (in `Robot`). See the other systems for example.